src/destination/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10.16.1-alpine
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY ./ ./
```

Comment:
best practice is to be more explicit here.

Reply:
I added a .dockerignore 🤔